### DISCUSSION TITLE
Templates

### DISCUSSION THREAD

**Discussion by maxverstappen** [maartentamboer/X-Touch-Mini-FS2020#73] [Ideas]
Does anyone have templates or an overlay  that cantoin the button layout for the default layout en the airbus layout?

**Comment by maxverstappen** [comment]
Maybe some one can use my first design (A4 landscape)
![Behringer image](https://user-images.githubusercontent.com/19385306/121388202-94233e80-c94b-11eb-96cc-09097ab7a861.png)

**Comment by maartentamboer** [comment]
taktility has a template file that can be used in illustrator: http://www.taktility.com/overlays/xtouch-mini/#Custom_print 
http://taktility.com/data/documents/X-Touch_Mini_Template.zip

**Reply by maartentamboer** [reply]
And there's someone who made a 3D printable overlay https://www.thingiverse.com/thing:4743655 
And people who made 3D printed knobs: https://www.thingiverse.com/thing:4702665 https://www.thingiverse.com/thing:4879975

**Comment by Delta-Charlie-DEV** [comment]
Hi,
Here's the template I made with Ms Publisher. you can freely edit as you want (change background colours, labels, and so on).

After printing, I use a laminator to improve durability and esthetic, then a cutter.... Great result !

![image](https://user-images.githubusercontent.com/76492436/121522373-55e25980-c9f5-11eb-8e0c-17808cb745f7.png)

[xtouch- A320.zip](https://github.com/maartentamboer/X-Touch-Mini-FS2020/files/6631233/xtouch-.A320.zip)
